department-of-veterans-affairs/va.gov-cms
# VACMS-6319: show, hide and require status details based on operating …

…status value

## Description

Closes #6319 

## Testing done

Visual / automated

## Screenshots
![Screen Shot 2022-05-30 at 7 26 39 PM](https://user-images.githubusercontent.com/21045418/171068064-a97f987c-8530-4894-ab96-2ba9f4b2c54f.png)

![Screen Shot 2022-05-30 at 7 28 35 PM](https://user-images.githubusercontent.com/21045418/171068076-89e2cddd-0c93-4c97-8881-49d2a7c792a9.png)

![Screen Shot 2022-05-30 at 7 28 57 PM](https://user-images.githubusercontent.com/21045418/171068089-7f2a54bd-556b-4eb1-bac6-b0109c5e5508.png)

## QA steps

As an admin user:

1. Visit /node/1104/edit
2. The operating status is currently set to normal services and hours
3. Confirm that the details field is not displayed
4. Change the operating status to Facility notice
5. Confirm that the details field is now displayed and marked as required
6. Do not supply any text for the details field and try to save a new published revision
7. Confirm the form doesn't submit and a notification is displayed
8. Put some content into the status details and try to submit again
9. Confirm the node does save
10. Edit the node again, set the status back to normal and save
11. Confirm the details field disappears
12. Edit the node again, set the status back to Facility notice
13. Confirm the details field reappears but is empty

### Select Team for PR review
